docs/samples/Microsoft.ML.Samples/Dynamic/DataOperations/LoadFromEnumerable.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
```

Comment:
Please add headers

Reply:
Do we need headers for samples? whatever we put here, will show up in the docs verbatim, so we didn't include the license header in any of the samples. .NET samples don't have headers either (example below). So I suggest not including headers for samples files.

[https://docs.microsoft.com/en-us/dotnet/api/system.io.file?view=netframework-4.8](https://docs.microsoft.com/en-us/dotnet/api/system.io.file?view=netframework-4.8)


---
In reply to: [282706294](https://github.com/dotnet/machinelearning/pull/3696#discussion_r282706294) [](ancestors = 282706294)

Reply:
i forgot, i'll remove them.